app/views/applications/ApplicationView.scala.html

```diff
@@ -366,10 +366,10 @@ <h3>IP allow list</h3>
                                 }
                             }
                         </td>
-                        <td data-collaborator-mfaEnabled>@(model.developers.find(dev => dev.email == collaborator.emailAddress).getOrElse(new User("","", "", None)).mfaEnabled match {
-                            case true => "Yes"
-                            case false => "No"
-                            })</td>
+                        <td data-collaborator-mfaEnabled>@(model.developers.find(dev => dev.email == collaborator.emailAddress) match {
+                            case Some(User(_, _, _, _, _, true)) => "Yes"
```

Comment:
How about `Some(user) if (user.mfaEnabled)` instead as otherwise working out what the true is is too much like hard work